.github/workflows/test.yml

```diff
@@ -61,22 +71,18 @@ jobs:
           export LD_LIBRARY_PATH=${R_HOME}/lib
           cargo test -- --nocapture --test-threads=1
 
-      # Installing 'msys2' on Windows only
-      - uses: msys2/setup-msys2@v2
-        if: runner.os == 'Windows'
-        with:
-          msystem: MINGW64
-          path-type: inherit
-          release: false
-          update: false
 
       - name: Build (Windows)
         if: runner.os == 'Windows'
         run: cargo build --target x86_64-pc-windows-gnu
-
+        env:
+          LIBCLANG_PATH: C:\msys64\mingw64\bin
```

Comment:
We shouldn't need `LIBCLANG_PATH` here, as this build doesn't generate bindings. The linking problem is that the wrong linker is being used, the one from Rtools rather than the one shipped by rustup. The same linking problem arises when the default host is `*-gnu` rather than `*-msvc`, but that doesn't seem to be the problem here.